KosherCocoa/Library/Core/Calendar/Chagim/KCJewishCalendar.m

```diff
@@ -762,7 +762,7 @@ - (nullable NSDate *)workingDateAdjustedForSunset
 {
     NSDate *returnDate = [super workingDate];
     
-    BOOL isAfterSunset = [[self sunset] timeIntervalSinceNow] < 0;
+    BOOL isAfterSunset = [returnDate timeIntervalSinceDate:returnDate] > 0;
```

Comment:
Good ol' KosherCocoa 😄
Wait, isn't the result of this method always 0? When would it be different than 0?

Reply:
Whoops. This is why I asked for code review. :p The arg on 765 needs to `self.sunset.`

Reply:
Fixed it, thanks for the review!